### PR TITLE
Make `RunParameters::test_start_time` a `DateTime`

### PR DESCRIPTION
**Pull request by ackintosh**
Changed `RunParameters::test_start_time` from String to DateTime for ease of use. 🙂

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.0] - unreleased
+### Change
+- Make `RunParameters::test_start_time` from `String` to `DateTime` for ease of use. See [PR 41].
```
Comment (mxinden) [inline CHANGELOG.md:9]:
```suggestion
- Change `RunParameters::test_start_time` from `String` to `DateTime` for ease of use. See [PR 41].
```

File: Cargo.toml [hunk 2]
```diff
@@ -6,11 +6,12 @@ edition = "2021"
 license = "Apache-2.0 OR MIT"
 name = "testground"
 repository = "https://github.com/testground/sdk-rust/"
-version = "0.4.0"
+version = "0.5.0"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+chrono = "0.4"
```
Comment (mxinden) [inline Cargo.toml:14]: Can we disable any of the default features for the sake of compile times? I.e. could this as well be:

```suggestion
chrono = { version = "0.4", default-features = false }
```

See https://github.com/chronotope/chrono/blob/main/Cargo.toml#L19

**Review by mxinden — COMMENTED** [review]
Thanks for the upstream improvement!

File: Cargo.toml (hunk 2 above)
Comment (ackintosh) [reply Cargo.toml]: I checked that the SDK with chrono default features disabled works fine. Thanks!

**Comment by ackintosh** [comment]
Thanks for your review! I added the `std` feature on https://github.com/testground/sdk-rust/pull/41/commits/caecbdcffaf0f28a61aeb29069da3448874ceaab due to [this error](https://github.com/testground/sdk-rust/actions/runs/3237954755/jobs/5305623493).

**Review by mxinden — APPROVED** [review]
:pray:

**Pull request merged**